building_energy_standards_data/database_files/support_constructions.json

```diff
@@ -8475,19 +8475,6 @@
         "material_5": null,
         "material_6": null
     },
-    {
-        "name": "U 1.22 SHGC 0.40 VT 0.44 Simple Glazing",
```

Comment:
@AmmarDeh - Do you know why this construction (and material) is being removed?

Reply:
@lymereJ This is related to the extra decimal comment above. I removed it because it corresponds to the construction with an extra decimal.